migrations/7_deploy_artists.js

```diff
@@ -0,0 +1,17 @@
+const Artists = artifacts.require('Artists');
+const ENSResolver = artifacts.require('ENSResolver');
+const FIFSRegistrar = artifacts.require('FIFSRegistrar');
+
+const newLabel = require('./helper/LoggedRegistration');
+
+module.exports = async (deployer, network, accounts) => {
+  await deployer.deploy(Artists, accounts[0], { from: accounts[0] });
```

Comment:
Do you want to do the governance trick with this? So governance controls artists?

Reply:
Probably out of scope but there needs to be some mechanism to add (trello task at low priority?)

Reply:
This is what I came here to say: what happens when the piece is buy an artist not yet in our system

Reply:
To governance and system to add, long term I agree but would rather spin off other tasks to reduce size of PR. I also don't think priority wise most important.